### PR TITLE
docs: upgrade docs from Docker Compose

### PR DESCRIPTION
**Pull request by jazzdan**
Skaffold is a TODO but I'm thinking it will follow this format.

### CODE REVIEW

**Comment by jazzdan** [comment]
Also I'll run `make docs` later, didn't want to muck up the diff with that for now.

File: docs/upgrade.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  spoonerisms:
+    build:
+      context: ./spoonerisms
+      dockerfile: ./spoonerisms/Dockerfile
+```
+
+In Tilt you tell us where your Dockerfile is and we infer your build context from that.
+
+```python
+  img = static_build("spoonerisms/Dockerfile", "gcr.io/myproject/spoonerisms")
```
Comment (nicks) [inline docs/upgrade.md]: should we talk about the `context` argument here?

File: docs/upgrade.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+```python
+def spoonerisms():
+  img = static_build("spoonerisms/Dockerfile", "gcr.io/myproject/spoonerisms")
+  yaml = read_file('spoonerisms.yaml')
+  service = k8s_service(img, yaml=yaml)
+  service.port_forward(9006)
+  return service
+```
+
+# From Skaffold
```
Comment (nicks) [inline docs/upgrade.md]: maybe we should make this a separate doc?

one of the big goals of these docs is to have metrics on what people are clicking on, so a separate page for Skaffold would help us know how often people are reading that doc

**Comment by nicks** [comment]
also, you'll want to add this the the TOC so that it shows up in the sidebar

File: docs/upgrade.md (hunk 1 above)
Comment (jazzdan) [reply docs/upgrade.md]: Since my docker compose config has `context` I think that's a good idea yeah.

**Pull request merged**